.github/workflows/ci-additional.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 2
-      - uses: ./.github/actions/detect-ci-trigger
+      - uses: keewis/ci-trigger@v1
```

Comment:
I'm assuming you will update this one too...

Reply:
thanks, I missed that one